pytest/run_tests_tls.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+HERE="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
```

Comment:
Can't your just use HERE=`${BASH_SOURCE[0]}` or do you need the full working directory?

Reply:
I prefer a full working folder.